flake.nix

```diff
@@ -33,12 +36,37 @@
           inherit system;
           overlays = [ overlay ];
         };
-      in
-      {
-        devShells.default = with pkgs; mkShell {
-          buildInputs = [
-            polykey-cli
-          ];
+      in rec {
+        devShells = {
+          # testing polykey-cli
+          polykey-cli = with pkgs; mkShell {
+            buildInputs = [
+              polykey-cli
+            ];
+          };
+          
+          # testing ignoreFromStore
```

Comment:
These comments don't make sense. In general comments should be capitalised. Our linter currently is only JS, but we should be standardized on all languages.

Reply:
If we get some time, @brynblack and I might be able to work on a formatter for nix. I know (and use) a nix LSP server, so it should be achievable for us to write a formatter. This can then be extended to various IDEs as needed.